designsafe/static/scripts/ng-designsafe/components/modals/data-browser-service-preview/data-browser-service-preview.component.js

```diff
@@ -53,6 +54,14 @@ class DataBrowserServicePreviewCtrl {
                                 const isGeoJson = validateGeoJson(body);
                                 if (isGeoJson) this.renderGeoJson(body);
                             }
+
+                            if (extension.includes('hazmapper')) {
+                                const body = JSON.parse(text);
```

Comment:
Since `body` is repeated here and on [line 51](https://github.com/DesignSafe-CI/portal/pull/867/files?file-filters%5B%5D=.html&file-filters%5B%5D=.js#diff-7de0ec5ac5c63c098f7efd05e0310be8a9e40b7913458cef7c2f527c07d98da0R51), consider pulling it out of both `if` stanzas

Reply:
I was concerned about other text formats throwing at parse. Would it be better to wrap it in a try catch with an empty catch? Or just add another or case for json formats?

Reply:
Good point! Makes sense to keep as is then.